ls-ui-react-app/package.json

```diff
@@ -3,30 +3,31 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@emotion/react": "^11.5.0",
     "@emotion/core": "^11.0.0",
+    "@emotion/react": "^11.5.0",
     "@testing-library/jest-dom": "^5.14.1",
     "@testing-library/react": "^11.2.7",
     "@testing-library/user-event": "^12.8.3",
     "@types/jest": "^26.0.24",
-    "@types/node": "^12.20.33",
-    "@types/react": "^17.0.31",
-    "@types/react-dom": "^17.0.10",
     "@types/jquery": "^3.5.8",
     "@types/jqueryui": "^1.12.16",
     "@types/lodash": "^4.14.176",
+    "@types/node": "^12.20.33",
+    "@types/react": "^17.0.31",
+    "@types/react-dom": "^17.0.10",
     "axios": "^0.24.0",
     "emotion": "^11.0.0",
-    "react": "^17.0.2",
     "jquery": "^3.5.8",
     "jquery-ui": "^1.13.0",
+    "lodash": "^4.17.21",
+    "query-string": "^7.0.1",
+    "react": "^17.0.2",
     "react-bootstrap": "^2.0.0",
+    "react-css-spinners": "^4.0.2",
```

Comment:
React-css-spinners was the only package I added. I'm not sure why my NPM modified the development environment like so? It appears that no files were actually changed, but the package.json file was modified and cluttering up the diff?